### PR TITLE
Network Connectivity Control

### PR DESCRIPTION
**Pull request by dilallkx**
**CHANGES:**
- General:
  - Bumped platform version to v1.6.8 (meepctl + repocfg + validator)
  - Upgraded demo1 & demo2 scenarios
  - Updated UT to test new features
- Data Model:
  - Moved connectivity config from Domain to Deployment in scenario
  - Added LADN flag in DNConfig to indicate if a DN is for local zone only
- Model Package:
  - Added Connectivity & DN Config validation
  - Added upgrade function to set default Connectivity Model to "OPEN" for existing & new scenarios
- Sandbox Controller:
  - Implemented connectivity endpoints to create/terminate PDU sessions
  - Uses PDU Session store package to add/remove PDU sessions in store
  - Sends message on local MQ to notify listeners of update to PDU sessions
- Net Char Manager:
  - Added message handler for PDU session updates
  - Updated Algorithm interface to accept current PDU sessions as input
  - Updated segment algorithm to consider PDU sessions while evaluating paths when scenario Connectivity model is "PDU"
- PDU Session Store package:
  - Renamed package to _meep-pdu-session-store_
  - Added methods to get all & delete all PDU sessions
- Frontend:
  - Support for Connectivity Model + Data Network configuration in element configuration pane
  - Default connectivity model is 'OPEN'
  - Default DNN for distant cloud nodes is 'internet'
  - Default DNN for EDGE/FOG nodes is 'edn'

**TESTING:**
- Cypress & UT pass

### CODE REVIEW

File: go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go [hunk 1]
```diff
@@ -1129,6 +1096,88 @@ func ceStopReplayFile(w http.ResponseWriter, r *http.Request) {
...
+
+	// Create PDU session
+	err = sbxCtrl.pduSessionStore.CreatePduSession(ueName, pduSessionId, &pduSessionInfo)
+	if err != nil {
+		log.Error(err.Error())
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	// Send message to inform other modules of terminated PDU session
```
Comment (roymx) [inline go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go:1132]: Editorial: comment should say of **created** PDU session

**Review by roymx — APPROVED** [review]
Well done, MG Manager changes were a bit hard to follow; I assume there was some level of refactoring.

File: go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go (hunk 1 above)
Comment (dilallkx) [reply go-apps/meep-sandbox-ctrl/server/sandbox-ctrl.go]: Fixed.
